src/chemistry/utils/tracer_data.F90

```diff
@@ -254,7 +254,7 @@ subroutine trcdata_init( specifier, filename, filelist, datapath, flds, file, &
     end if
 
     if (masterproc) then
-       write(iulog,*) 'trcdata_init: data type: '//trim(data_type)//' file: '//trim(filename)
+       write(iulog,'(a)') 'trcdata_init: data type: '//trim(data_type)//' file: '//trim(filename)
```

Comment:
This would be a lot more efficient as 
```suggestion
       write(iulog,'(4a)') 'trcdata_init: data type: ', trim(data_type), ' file: ', trim(filename)
```